testing/coverage.rst

```diff
@@ -258,8 +258,9 @@ times.
 
 Filing the Issue
 ================
-Once you have increased coverage, you need to create an issue on the
-`issue tracker`_ and submit a :ref:`pull request <pullrequest>`.
+Once you have increased coverage,
+you need to create an issue on the `issue tracker`_ and
+submit a :ref:`pull request <pullrequest>`.
```

Comment:
This is another workaround: by reflowing the paragraph I avoid having the link and the `ref` on the same line.